packages/front-end/components/Settings/Team/InviteModal.tsx

```diff
@@ -55,6 +58,20 @@ const InviteModal: FC<{ mutate: () => void; close: () => void }> = ({
 
   // Hit their free limit and needs to upgrade to invite more team members
   if (showUpgradeModal) {
+    // The upgrade modal won't actually show for these plans, so show a generic failure / upgrade message
+    if (["pro", "pro_sso", "enterprise"].includes(accountPlan ?? "")) {
```

Comment:
I'm a bit confused about the wording here—it says the upgrade modal won't show for these plans, but it's in the block `showUpgradeModal` which is a bit confusing to understand just from reading the comments and code.